InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -28,6 +28,7 @@
     public boolean feeding = false;
     public boolean feederVelocityControl = false;
     public boolean shooterVelocityControl = false;
+    public static boolean upToSpeed = false;

```

Comment:
This shouldn't be static. It should be an instance variable of the ShooterSubsystem. It should have a getter.